src/components/App/App.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+import Form from '../Form/Form'
+// import logo from './logo.svg';
+import './App.css';
+
+const App = () => {
+
+  const [name, setName] = useState('')
+  const [username, setUsername] = useState('')
+  const [email, setEmail] = useState('')
+  const [page, setPage] = useState(0)
+  const [make, setMake] = useState('')
+  const [model, setModel] = useState('')
+  const [year, setYear] = useState('')
+  const [origin, setOrigin] = useState('')
+  const [destination, setDestination] = useState('')
+
+  return (
+    <div className="App">
+      {page === 0 && 
+        <Form 
+          header="name"
+          page={page}
+          setPage={setPage}
+          inputs={[{property: name, method: setName, placeholder: 'name'}, 
+          {property: username, method: setUsername, placeholder: 'username'}, 
+          {property: email, method: setEmail, placeholder: 'email'}]} 
+        />
+      }
+      {page === 1 &&
+        <Form 
+          header="car"
+          page={page}
+          setPage={setPage}
+          inputs={[{property: make, method: setMake, placeholder: 'make'}, 
+          {property: model, method: setModel, placeholder: 'model'}, 
+          {property: year, method: setYear, placeholder: 'year'}]} 
+        />
+      }
+      {page === 2 &&
+        <Form 
+          header="origin and destination"
+          page={page}
+          setPage={setPage}
+          inputs={[{property: origin, method: setOrigin, placeholder: 'origin'}, 
+          {property: destination, method: setDestination, placeholder: 'destination'}]} 
+        />
+      }
+      {page === 3 &&
+        <Form 
+          header="days and time"
+          page={page}
+          setPage={setPage}
+          inputs={[{property: destination, method: setDestination, placeholder: 'destination'}]} 
+        />
+      }
+    </div>
+  );
+}
+
```

Comment:
I like this setup... we talked a little bit about it already but we may eventually want to have all this conditional rendering happen in another component that contains all forms. 

We would have to pass down all the state and methods, which is a lot. OR we can go wild and mess with the context api.